core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -107,11 +125,14 @@ object ConsumerPerformance {
 
   }
 
-  private[tools] def printHeader(showDetailedStats: Boolean): Unit = {
-    if (showDetailedStats)
-      println("start.time, end.time, data.consumed.in.MB, MB.sec, data.consumed.in.nMsg, nMsg.sec")
-    else
-      println("time, threadId, data.consumed.in.MB, MB.sec, data.consumed.in.nMsg, nMsg.sec")
+  private[tools] def printHeader(showDetailedStats: Boolean, useOldConsumer: Boolean): Unit = {
+    if (!showDetailedStats) {
+        println("start.time, end.time, data.consumed.in.MB, MB.sec, data.consumed.in.nMsg, nMsg.sec" +
+          s"${if (!useOldConsumer) ", rebalance.time.ms, fetch.time.ms, fetch.MB.sec, fetch.nMsg.sec" else ""}"
+        )
+      } else {
+        println("time, threadId, data.consumed.in.MB, MB.sec, data.consumed.in.nMsg, nMsg.sec")
```

Comment:
In the detailed view, can we safely think `fetch.time.ms` is always equal to `elapsedMs` in `printProgressMessage`? If yes, then the new fields `fetch.MB.sec` and `fetch.nMsg.sec` will always be equal to the existing ones `MB.sec` and `nMsg.sec` respectively, so that's why I did not have those new fields included in the detailed view. Does it make any sense?